Header header
double Acceleration_X
double Acceleration_Y
double Rate_Z